README.md

```diff
@@ -107,14 +107,14 @@ theme = "dario"
   description = "A description of your site" # a description of your site that will be used in the meta tags
 ```
 
-To add intro text to your home page, create a file at `content/_intro.md` with contents similar to the following:
+To add intro text to your home page, create a file at `content/_index.md` with contents similar to the following:
 
-```toml
-+++
-headless = true
-title = "Home Page Text" # the text to display on the "/" homepage
-description = "A minimal web log." # The description of the home page that will be used in the open graph meta tags
-+++
+```yaml
+---
+[params.homeInfo]
+title: "Home Page Text" # the text to display on the "/" homepage
+description: "A minimal web log." # The description of the home page that will be used in the open graph meta tags
```

Comment:
`description` here is not really needed, to be honest. It can be moved above:

```yaml
---
description: "A minimal web log." # The description of the home page that will be used in the open graph meta tags

[params.homeInfo]
title: "Home Page Text" # the text to display on the "/" homepage
```

This makes it the page description and then the extra code from `opengraph.html` can be removed.

If we do that, we can also rename `homeInfo.title` to `subtitle` or something and simplify the code further.

But I'd have to test this all out so I haven't included it in this patch.

Reply:
> I haven't included it in this patch

Done with https://github.com/GrantBirki/dario/pull/39/commits/c4cf58974d829cfed8b40a660ff7f2af584e249c